src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpEndPointManager.cs

```diff
@@ -96,10 +96,7 @@ private static void AddPrefixInternal(string p, HttpListener listener)
             if (lp.Host != "*" && lp.Host != "+" && Uri.CheckHostName(lp.Host) == UriHostNameType.Unknown)
                 throw new HttpListenerException((int)HttpStatusCode.BadRequest, SR.net_listener_host);
 
-            if (lp.Path!.Contains('%'))
-                throw new HttpListenerException((int)HttpStatusCode.BadRequest, SR.net_invalid_path);
-
-            if (lp.Path.IndexOf("//", StringComparison.Ordinal) != -1)
```

Comment:
Is it right here? "//" and '/' looks like different.

Reply:
Nope, this is a mistake.